doc/src/manual/arrays.md

```diff
@@ -747,15 +750,34 @@ sparse data structure, including structural nonzeros. In order to count the exac
 values that are nonzero, use [`countnz()`](@ref), which inspects every stored element of a sparse
```

Comment:
While cleaning this up, perhaps "the exact number of actual values that are nonzero" should be "the exact number of numerical nonzeros" or "the exact number of numerical nonzero values"?